src/py/mat3ra/made/tools/modify.py

```diff
@@ -500,3 +500,15 @@ def displace_interface(
     new_material.to_crystal()
     new_material = wrap_to_unit_cell(new_material)
     return new_material
+
+
+def get_interface_part(
```

Comment:
Maybe analyze would fit better, why modify?

Reply:
It returns a material, while analyze means we get some properties from the material.